### ISSUE TITLE
docker:push - unexpected behavior

### ISSUE THREAD

**Issue by knoppiks** [rhuss/docker-maven-plugin#165]
I specified a docker push action on maven deploy with:
```xml
<execution>
    <id>push</id>
    <phase>deploy</phase>
    <goals>
        <goal>push</goal>
    </goals>
</execution>
```
For the image `<name>dreg.private-server.local/image-name</name>`. 
I executed a docker push with `docker push dreg.private-server.local/image-name` which succeeded.

But the maven execution fails with 
```
[ERROR] Failed to execute goal org.jolokia:docker-maven-plugin:0.11.3:push (push) on project image-name-war: Unable to push image [dreg.private-server.local/image-name] to registry [null] : EOF (Internal Server Error: 500) -> [Help 1]
```

Maybe I'm wrong but I expected the maven goal to do exactly the same command I issued by hand, as I defined the name of the image with a registry prefix.

**Comment by rhuss** [comment]
Indeed, it should work the same. Could you run your example with -X ? Could you try also to provide a `-Ddocker.registry=dreg.private-server.local` on the command line (maybe even with an explicite port `:5000` appened, too ?)

thanks ...

**Comment by jgangemi** [comment]
most likely you are hitting the issue where you need to specify a dummy username/password in the auth config section, see #102.

**Comment by rhuss** [comment]
BTW, #102 is fixed in the next release 0.11.4 which will be out at the end of this week.

**Comment by knoppiks** [comment]
Thanks for the support, the issue was indeed #102.
After adding the empty <authConfig> the push succeeds.

Changing to v0.11.4 won't do the trick somehow, I guess there are other problems with my configuration.
